guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<p>Is a gif missing here for dilation? like in : </p><p><a href="https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5" target="_blank">https://towardsdatascience.com/review-dilated-convolution-semantic-segmentation-9d5a5bd768f5</a></p><p> </p><p>Also labeling GIFs as (<strong>Standard Convolution (l=1)) and Dilated Convolution (l&gt;1)</strong> will improve readability of the guide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I don't see any gif missing here.</p>

Reply:
<p>I don't see any gif missing here.</p>